tests/unit/mock_utils.py

```diff
@@ -0,0 +1,5 @@
+# For Python 3.5, Mock's call list is not ordered
```

Comment:
could you expand on what lead to this conclusion? seems like a major bug in their library if this is the case which should have been fixed w/ a package version bump.

Reply:
It is not Mock's call list. I came up with this utils before I realized that this is not just Mock's call list, but `dict` implementation: https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6
After many failed attempted to fix Python 3.5 build, Python 3.6. and 3.7 were always passed.  I thought that the problem is Mock's call list. So I wrote this utils. After I realized it's `dict` that unordered in Python 3.5, I forgot to revert this.